migration/requirements.txt

```diff
@@ -0,0 +1,5 @@
+nameparser
```

Comment:
minor: This is probably the only dependency that has to be moved to the `Pipfile`. `psycopg` should be part of `invenio_rdm_migrator`